util/feed.go

```diff
@@ -91,6 +91,9 @@ type Feed struct {
 // all Subscribers to the feed. Events published to a closed feed, or to a feed
 // with no Subscribers, will be ignored.
 func (f *Feed) Publish(event interface{}) {
+	if f == nil {
```

Comment:
Why get rid of this?  In my mind, the feed has valid nil behavior; the behavior of a nil feed is the same as a non-nil feed with zero subscribers.